tests/blockchain/regtest/SimpleTransfer.cpp

```diff
@@ -88,18 +68,20 @@ TEST_F(Regtest_fixture_simple, DISABLED_send_to_client)
          number_of_test++) {
 
         SendCoins(*receiver, *sender, target_height, coin_to_send);
-        std::this_thread::sleep_for(std::chrono::seconds(20));
 
         auto loaded_transactions = CollectTransactionsForFeeCalculations(
             *sender, send_transactions_, transactions_);
         auto fee = CalculateFee(send_transactions_, loaded_transactions);
         send_transactions_.clear();
 
-        EXPECT_EQ(GetBalance(*sender), sender_amount - coin_to_send - fee);
-        EXPECT_EQ(GetBalance(*receiver), receiver_amount + coin_to_send);
+        sender->expected_balance_ -= Amount{coin_to_send} + fee;
+        receiver->expected_balance_ += coin_to_send;
```

Comment:
we do not need to Add Amount{coin_to_send} ? instead just coin_to_send ?

Reply:
Right now no because I have error "error: no match for ‘operator+’".  I am not sure why line 78 work probably if i add one value Amount constructor call implicitly. I can leave it like this or write operator to Amount class.